README.md

```diff
@@ -1,34 +1,75 @@
-[![Build Status](https://travis-ci.org/ServiceInnovationLab/fsd-spike.svg?branch=master)](https://travis-ci.org/ServiceInnovationLab/fsd-spike)
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ServiceInnovationLab/fsd-spike.png?columns=all)](https://waffle.io/ServiceInnovationLab/fsd-spike?utm_source=badge)
+# Family Services Directory Searchinator
+[![Build Status](https://travis-ci.org/ServiceInnovationLab/FamilyServiceDirectorySearchinator.svg?branch=master)](https://travis-ci.org/ServiceInnovationLab/FamilyServiceDirectorySearchinator)
+[![Waffle.io - Columns and their card count](https://badge.waffle.io/ServiceInnovationLab/FamilyServiceDirectorySearchinator.png?columns=all)](https://waffle.io/ServiceInnovationLab/FamilyServiceDirectorySearchinator?utm_source=badge)
 
-This runs [GH-Pages](https://serviceinnovationlab.github.io/fsd-spike/)
+This runs [on github pages](https://serviceinnovationlab.github.io/FamilyServiceDirectorySearchinator/)
 
-## Deployment
+## Overview
+A React frontend for the existing Family Services Directory providing keyword, category, and location-based search results
 
-To deploy to github pages, checkout the branch you want to deploy and then run
+## Environments
+**Environment** | **URL**  | **Git Branch**
+--- | --- | ---
+Github Pages | https://serviceinnovationlab.github.io/FamilyServiceDirectorySearchinator/ | master |
+
+## Project Resources
+
+**Resource** | **URL**
+--- | ---
+Backlog | https://waffle.io/ServiceInnovationLab/FamilyServiceDirectorySearchinator
+CI | https://circleci.com/gh/ServiceInnovationLab/fsd-spike
+
+**Role(s)** | **Name(s)**
+--- | ---
+Team | Family Services Directory Searchinator
+Developers | [Brenda Wallace](https://github.com/Br3nda), [Dana Iti](https://github.com/dlouise64), [Jacob Ong](https://github.com/JacOng17), [Lyall Morrison](https://github.com/lamorrison), [Mischa Saunders](https://github.com/mischa-s)
+Designers |
+Testers | 
+Project Manager |
+Product Owner |
+
+## Comms
+Slack: LabPlus-team #family-searchinator
+
+## Setup
+
+## Development
+
+to install dependencies, cd into folder and run
 ```
-yarn run deploy
+npm install
```

Comment:
Can you include the reason for this change ?

Reply:
accidentally changed it when copy+pasting from Rapu Ture, I'll review the instructions